example/README.md

```diff
@@ -0,0 +1,47 @@
+# Plutus example project
+
+## NixOS
+
+This directory also includes a `configuration.nix` file which can be used in a NixOS installation. This makes sure the correct system libraries are available as well as adding the Haskell IDE Engine cachix nix cache. This `configuration.nix` file can be used on a clean NixOS installation and everything will just work.
```

Comment:
The "also" is a bit weird as this is the first thing.

Reply:
I'm going to move it below `##Getting Started` and add a little blurb, see the next commit.